### ISSUE TITLE
Getting a list of public stories for a user

### ISSUE THREAD

**Issue by david-meza** [Medium/medium-api-docs#51]
I'm interested in accessing the data that is here https://medium.com/me/stories/public but I did not find any documentation for it. This seemed a bit stranger because you can post a story on behalf of a user, but not get their existing stories. Is this currently supported by the API?

**Comment by david-meza** [comment]
I guess I found my answer in #30

**Comment by SpenUK** [comment]
+1 It'd be nice to see some of the openness that wordpress offer, though I'm sure there are business decisions around this.

**Comment by danielfriis** [comment]
👍

**Comment by mikes01** [comment]
+1

**Comment by Aqmhkf** [comment]
İ just came back

**Comment by Aqmhkf** [comment]
Ok right person

**Comment by Aqmhkf** [comment]
Thankssss you

**Comment by Aqmhkf** [comment]
Ahhhhh

**Comment by jleepeak** [comment]
+1

**Comment by benmedium** [comment]
We don't support this in the API right now, but have you considered using the profile RSS feeds?

**Comment by mikes01** [comment]
Do you have idea how to filter articles from comments?

**Comment by crespum** [comment]
@benmedium I'm interested in this too, but I also need the tags associated to a post, so RSS is not enough.

**Comment by mikes01** [comment]
At response from https://medium.com/@ev/latest?format=json I get weird values in "createdAt" fields e.g. "createdAt":1469697040721 . It is not unix timestamp. Do you know how i can convert it to date?

**Comment by ambethia** [comment]
https://medium.com/@ambethia/latest?format=json

Something tells me that this isn't supposed to be there?

```
])}while(1);</x>
```

**Comment by benmedium** [comment]
The JSON page is not intended to be used as a read API. The extra code is there to support our own use and is a standard technique to avoid JSON hijacking.

Also, it's worth adding that feeds do now contain tags - using them is highly recommended!

**Comment by evaristoc** [comment]
@benmedium 
Thanks for your previous answers to this thread. You recently wrote:
>The JSON page is not intended to be used as a read API.

Might I ask if there is a way to get ALL titles written by specific authors? I also found that all titles written by a person might not be not listed by using conventional methods of exploring the `.../feed/...` page: only the few  latest ones.

**Comment by elwayman02** [comment]
Yea, we definitely need an API that has more information than a simple RSS feed. I basically can't justify switching to Medium as my publishing platform until this is supported.

**Comment by ambethia** [comment]
Yup, the RSS feed also include things like comments left on other posts which makes it a little difficult to just, say, put links to recent blog posts on another site, which is what my students were trying to do. There might be another way to go around that, but setting up authentication just to do that seemed like overkill.

**Comment by damko** [comment]
+1 and please clarify better in the current API that Publications are not user's public posts. I've got tricked by the current documentation.

**Comment by devlead** [comment]
@ambethia agree RSS is a little bit too, heavy and it would be nice to filter to only my stories and not my responses to other people's stories. I do list only "mine" on my web page by filtering on tags, basically I don't tag my responses, but I always tag my stories, so my work around is to only list the posts with tags.
Personally I would prefer API methods for this, before finding this issue I suggested that in Medium/medium-api-docs#48

**Comment by markfasel** [comment]
Hi there, I have built a basic function using AWS Lambda and AWS API Gateway if anyone is interested. A detailed explanation is found on [this blog post here](https://medium.com/@markfasel/building-a-basic-web-service-to-display-your-medium-blog-posts-on-your-website-using-aws-api-48597b1771c5) and the repository for the the Lambda function built with Node.js is found here [https://github.com/markfasel/medium-nodejs-lambda](url)

**Comment by devlead** [comment]
@markfasel great tip & post 👍 Did similar as an Azure Function using the XML from RSS feed, explained in [this blog post](https://hackernoon.com/bring-static-to-life-using-serverless-e722be5eaf84) and the repository for the function is built with C# can be found here https://github.com/azurevoodoo/BreathLifeIntoStaticSiteWithAzureFunctions.git

Lamda / Azure functions perfect for this kinda thing, still proper API would save a few bytes 😉 and feel less hacky.

**Comment by markfasel** [comment]
@devlead thanks! I just read yours. Good post. I'll keep that in mind if I build something with C#! Thanks again!

**Comment by enginebai** [comment]
Hi, I've built [PyMedium - Unofficial Medium API](https://github.com/enginebai/PyMedium) to provide developers a easier way to access data from Medium.

You can use `/@{username}/posts` to get the public posts from the user.
Star and contribution are welcome.

**Comment by theholy7** [comment]
@mikgry - I hope you found that createdAt was a unix timestamp in milisecond. if you divide by 1000 you get the correct values.

**Comment by idoco** [comment]
@ambethia Totally unrelated, I just wanted to comment about the `])}while(1);</x>` you mentioned. This a known trick to battle XSS attacks by trapping the executed malicious code in an endless `while(1)` loop;

**Comment by newhouse** [comment]
Yes, seconding what @idoco said, @ambethia 

See this Issue for more details on that bad looking JSON: https://github.com/Medium/medium-api-docs/issues/101

**Comment by terencechow** [comment]
So is there no way to filter out comments? Seems like a feed is very useless if you get a bunch of random responses without context from the people you are following...

**Comment by GitMoVA** [comment]
+1

Any way to filter out comments?

**Comment by darklow** [comment]
Looking for a way to get user posts only in RSS without responses too. I used to filter out the ones where categories are not used, but some authors use tagging even for comments, so that doesn't work anymore.
